app/src/main/java/com/github/polybooks/utils/InternetPrice.kt

```diff
@@ -0,0 +1,41 @@
+package com.github.polybooks.utils
+
+import com.github.polybooks.core.ISBN
+import com.google.gson.JsonParser
+import java.io.InputStream
+import java.io.InputStreamReader
+import java.net.HttpURLConnection
+import java.net.URL
+import java.util.concurrent.CompletableFuture
+
+val DOLLAR_TO_CHF: Double = 0.9
+val INTERNET_PRICE_UNAVAILABLE: String = "__UNAVAILABLE__"
+
+fun getInternetPrice(isbn : ISBN): CompletableFuture<String> {
+    return CompletableFuture.supplyAsync {
+        val appId = "?key=1orhhq0s3nrtl4kf93r3"
```

Comment:
Well, this exposes an API key

Reply:
@Snowflake8

Reply:
To much work to do otherwise for an obscure website key